### PR TITLE
Update error messages

### PR DESCRIPTION
**Pull request by delaaxe**

### CODE REVIEW

**Review by Leonard-Pat — APPROVED** [review]

File: contracts/argent_account.cairo [hunk 1]
```diff
@@ -24,9 +24,9 @@ mod ArgentAccount {
     #[external]
     fn initialize(signer: felt, guardian: felt, guardian_backup: felt) {
         // check that we are not already initialized
-        assert(signer::read() == 0, 'argent: already initialized');
+        assert(signer::read() == 0, 'argent/already-initialized');
         // check that the target signer is not zero
-        assert(signer != 0, 'argent: signer cannot be null');
+        assert(signer != 0, 'argent/null-signer');
```
Comment (delaaxe) [inline contracts/argent_account.cairo:29]: question for @gaetbout here: it seems the name `signer` refers to either the storage variable or the function argument depending on how it's used. is this how it works in rust too?

this seems a bit dangerous to me, should we prefix all our function arguments with `_` like we do in solidity?

**Review by gaetbout — APPROVED** [review]

File: contracts/argent_account.cairo (hunk 1 above)
Comment (delaaxe) [reply contracts/argent_account.cairo:29]: I'd rather keep storage as is and rename method args

Comment (juniset) [reply contracts/argent_account.cairo:29]: We can simply use `new_signer` for the argument if that makes it easier.

**Review by juniset — APPROVED** [review]

File: contracts/argent_account.cairo (hunk 1 above)
Comment (delaaxe) [reply contracts/argent_account.cairo:29]: I'll prefix the arguments with `initial_`

Comment (gaetbout) [reply contracts/argent_account.cairo:29]: I thin new_XXX makes more sense, initial_XXX makes me think that it is the initial value that needs to be changed

Comment (delaaxe) [reply contracts/argent_account.cairo:29]: fine then. there's also an unused `guardian_backup` arg that I'll remove

**Pull request merged**